rest-adapter/src/main/java/uk/gov/justice/services/adapter/rest/envelope/RestEnvelopeBuilder.java

```diff
@@ -114,10 +115,10 @@ private Metadata buildMetadata() {
         if (contains(USER_ID, httpHeaders) || contains(SESSION_ID, httpHeaders)) {
             JsonObjectBuilder contextBuilder = Json.createObjectBuilder();
             if (contains(USER_ID, httpHeaders)) {
-                contextBuilder = contextBuilder.add(Metadata.USER_ID, getHeader(USER_ID, httpHeaders));
+                contextBuilder = contextBuilder.add(JsonObjectMetadata.USER_ID, getHeader(USER_ID, httpHeaders));
             }
             if (contains(SESSION_ID, httpHeaders)) {
-                contextBuilder = contextBuilder.add(Metadata.SESSION_ID, getHeader(SESSION_ID, httpHeaders));
+                contextBuilder = contextBuilder.add(JsonObjectMetadata.SESSION_ID, getHeader(SESSION_ID, httpHeaders));
```

Comment:
We have HeaderConstants already statically imported. Some of these constants are common.